pkg/apis/servicecatalog/types.go

```diff
@@ -270,10 +270,17 @@ type InstanceSpec struct {
 	// provisioned from.
 	PlanName string
 
-	// Parameters is a YAML representation of the properties to be
+	// Parameters is a set of the parameters to be
 	// passed to the underlying broker.
+	// The inline YAML/JSON payload to be translated into equivalent
+	// JSON object.
+	// +optional
 	Parameters *runtime.RawExtension
 
+	// List of sources to populate parameters.
```

Comment:
Comments don't match between this and binding, you should ensure the same level of detail is taken in both places.